debezium-core/src/test/java/io/debezium/transforms/ExtractNewRecordStateTest.java

```diff
@@ -294,6 +304,34 @@ public void testAddSourceField() {
         }
     }
 
+    @Test
+    public void testAddTopicRoutingField() {
```

Comment:
Could you add a test to make sure that the new column value is applied in case of an update event?

Reply:
Done